### ISSUE TITLE
Duplicate files (normal + patched) are listed only once

### ISSUE THREAD

**Issue by 20Phoenix** [digableinc/tinfoil#33]
Now I notice something else:
I've two versions of an update, the normal one and a patched one (new update for use without 6.2.0).
Names are similar, the patched one has "patched" added in the name.
But only one of these two is listed at New Updates (after pressing right stick) :(
Would be nice to have them both listed.

**Comment by blawar** [comment]
It is designed to be indexed on the title id, it doesnt handle duplicate titles unless you install them through the file browser.
